CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
     * Remove `PaymentButtonColor` `.black`, `.silver`, `.blue`, and `.darkBlue`
     * Add `PaymentButtonColor.gold` for `PayPalCreditButton`
     * Remove `PaymentButtonSize` cases: `.miniWithWordmark`, `.collapsed`, `.expanded`, `.full`
+    * Rename `PayPalButton.Representable` to `PayPalButtonView`, `PayPalPayLaterButton.Representable` to `PayPalPayLaterButtonView`, and `PayPalCreditButton.Representable` to `PayPalCreditButtonView` for SwiftUI
```

Comment:
Not sure if this change should be reverted and done in a separate PR based on this comment: https://github.com/paypal/paypal-ios/pull/247#discussion_r1484672567

Reply:
I'll be adding in `.Representable` again so this log will be removed